cmd/injective-price-oracle/oracle.go

```diff
@@ -187,26 +197,50 @@ func oracleCmd(cmd *cli.Cmd) {
 					return nil
 				}
 
-				dynamicFeedConfigs = append(dynamicFeedConfigs, feedCfg)
+				feedConfigs = append(feedConfigs, feedCfg)
 
 				return nil
 			})
 
 			if err != nil {
-				err = errors.Wrapf(err, "dynamic feeds dir is specified, but failed to read from it: %s", *dynamicFeedsDir)
+				err = errors.Wrapf(err, "feeds dir is specified, but failed to read from it: %s", *feedsDir)
 				log.WithError(err).Fatalln("failed to load dynamic feeds")
 				return
 			}
 
-			log.Infof("found %d dynamic feed configs", len(dynamicFeedConfigs))
+			log.Infof("found %d dynamic feed configs", len(feedConfigs))
+		}
+
+		var storkFetcher oracle.StorkFetcher
+
+		if feedConfigs != nil {
+			for _, feedCfg := range feedConfigs {
+				if feedCfg.ProviderName == oracle.FeedProviderStork.String() {
+					conn, err := pipeline.ConnectWebSocket(ctx, *websocketUrl, *websocketHeader, oracle.MaxRetriesReConnectWebSocket)
```

Comment:
isn't this connecting once per config file?

we could have a bool var in the `filepath.WalkDir` that we set to true if there are any stork config files and then init afterwards